coderd/mcp_http.go

```diff
@@ -11,7 +12,15 @@ import (
 	"github.com/coder/coder/v2/codersdk"
 )
 
+type MCPToolset string
+
+const (
+	MCPToolsetStandard MCPToolset = "standard"
```

Comment:
We might define `Standard` one to be the default empty value, then we don't need to do additional checks later on.

```suggestion
	MCPToolsetStandard MCPToolset = ""
```

Reply:
Neat trick, but it feels unintuitive - especially if we add debug logging with toolset names later. I'd be surprised to see this pattern as a reader.